kubemarine/resources/configurations/compatibility/internal/plugins.yaml

```diff
@@ -30,29 +20,16 @@ calico:
   v1.25.2:
     version: v3.24.2
   v1.25.7:
-    version: v3.24.2
+    version: v3.26.1
   v1.26.3:
-    version: v3.24.2
+    version: v3.26.1
```

Comment:
Plugin versions were increased, but they are not scheduled for upgrade. This is strange enough, probably you increased them manually? Please refer to the comment in the header of the file
> !!!Do not change the content of the file!!!